static/src/js/containers/DownloadHistoryContainer/__tests__/DownloadHistoryContainer.test.js

```diff
@@ -1,89 +1,108 @@
+import nock from 'nock'
 import React from 'react'
-import Enzyme, { shallow } from 'enzyme'
-import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
-
-import actions from '../../../actions'
-import {
-  DownloadHistoryContainer,
-  mapDispatchToProps,
-  mapStateToProps
-} from '../DownloadHistoryContainer'
-import { DownloadHistory } from '../../../components/DownloadHistory/DownloadHistory'
+import { render, screen } from '@testing-library/react'
+import { MemoryRouter } from 'react-router-dom'
+import { Provider } from 'react-redux'
+import configureStore from 'redux-mock-store'
 
-Enzyme.configure({ adapter: new Adapter() })
+import DownloadHistoryContainer, {
```

Comment:
We don't test the default exports of our containers, we test the named export. We do this in every container test so that we don't have to setup a full redux store, we just give it the props we want and test our own code

```
import {
  DownloadHistoryContainer,
  mapDispatchToProps,
  mapStateToProps
} from '../DownloadHistoryContainer'
```